android/src/main/java/io/mosip/tuvali/transfer/Util.kt

```diff
@@ -37,19 +33,41 @@ class Util {
            |  firstByte | secondByte  |
            |   (byte 0) |  (byte 1)   |
            +------------+-------------+
-    secondByte.toInt() + (256 * firstByte.toInt())
    */
 
-    @OptIn(ExperimentalUnsignedTypes::class)
-    fun twoBytesToIntBigEndian(num: ByteArray): Int {
-      val convertedNum = num.toUByteArray()
-      val firstByte: UByte = convertedNum[0]
-      val secondByte: UByte = convertedNum[1]
-      return secondByte.toInt() + (256 * firstByte.toInt())
+    fun intToByteArray(num: Int, size: ArraySize): ByteArray {
+      val capacity = if (size == TwoBytes) 2 else 4
```

Comment:
We can avoid this line, by enriching the enum like this 

```kotlin
enum class ArraySize(val size: Int) {
    FourBytes(4),
    TwoBytes(2);
}

val arrSize = ArraySize.FourBytes
println("size: ${arrSize.size}")

// prints size: 4
```

So capacity can be derived with `size.size`.